aws-creds/iam.tf

```diff
@@ -10,6 +10,11 @@ terraform {
 provider "aws" {
   version = ">= 2.28.1"
   region  = var.region
+#    assume_role {
```

Comment:
is this accidental debugging stuff we should remove?

Reply:
I want it in there, because I might want to template from that.  Will remove eventually